StreamNode/Services/Settings/SettingsService.cs

```diff
@@ -0,0 +1,50 @@
+﻿using Config.Net;
+using System.IO;
+using Newtonsoft.Json;
+
+namespace StreamNode.Services.Settings
+{
+    public class SettingsService
+    {
+        private const string _HTTP_SERVER_IP = "0.0.0.0";
+        private const int _HTTP_SERVER_PORT = 0;
+        private const int _WEBSOCKET_PORT = 0;
+        // TODO add the other default settings
+
+        private string PATH = "config.json";
+        public ISettings settings { get; set; }
+
+        public SettingsService()
+        {
+            Init();
+            GetSettings();
+        }
+
+        private void Init()
+        {
+            if(!File.Exists(this.PATH))
```

Comment:
We don't need a new instance because of defaultvalue annotation
https://github.com/aloneguid/config#default-values